sdk/netapp/Microsoft.Azure.Management.NetApp/src/Generated/Models/SnapshotPolicy.cs

```diff
@@ -40,22 +40,20 @@ public SnapshotPolicy()
         /// <param name="name">Resource name</param>
         /// <param name="type">Resource type</param>
         /// <param name="tags">Resource tags</param>
-        /// <param name="name1">Snapshot policy name</param>
         /// <param name="hourlySchedule">hourlySchedule</param>
         /// <param name="dailySchedule">dailySchedule</param>
         /// <param name="weeklySchedule">weeklySchedule</param>
         /// <param name="monthlySchedule">monthlySchedule</param>
         /// <param name="enabled">The property to decide policy is enabled or
         /// not</param>
         /// <param name="provisioningState">Azure lifecycle management</param>
-        public SnapshotPolicy(string location, string id = default(string), string name = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), string name1 = default(string), HourlySchedule hourlySchedule = default(HourlySchedule), DailySchedule dailySchedule = default(DailySchedule), WeeklySchedule weeklySchedule = default(WeeklySchedule), MonthlySchedule monthlySchedule = default(MonthlySchedule), bool? enabled = default(bool?), string provisioningState = default(string))
+        public SnapshotPolicy(string location, string id = default(string), string name = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), HourlySchedule hourlySchedule = default(HourlySchedule), DailySchedule dailySchedule = default(DailySchedule), WeeklySchedule weeklySchedule = default(WeeklySchedule), MonthlySchedule monthlySchedule = default(MonthlySchedule), bool? enabled = default(bool?), string provisioningState = default(string))
```

Comment:
This is a breaking change: removing `string name1 = default(string)` param.

Has this breaking change been approved?

Reply:
We removed name property from snapshotpolicyProperties it is a violation of RPC .
https://armwiki.azurewebsites.net/api_contracts/RPCstrongvalidation.html

We have seen ICMs raised for this
`Resource response returned by provider 'Microsoft.NetApp' a JSON payload that contained invalid properties at the root scope, the invalid properties are 'Name'.`

https://portal.microsofticm.com/imp/v3/incidents/details/209491961/home